packages/markdown-docx/src/OOXMLTransformer/index.js

```diff
@@ -58,171 +87,178 @@ class OoxmlTransformer {
     }
 
     /**
-     * Gets the value of the variable
+     * Constructs a ciceroMark Node JSON from the information
      *
-     * @param {Array} elements the variable elements
-     * @returns {string} the value of the variable
+     * @param {object} nodeInformation Contains properties and value of a node
+     * @return {object} CiceroMark Node
      */
-    getValue(elements) {
-        const variableContents = elements[1];
-        for (const property of variableContents.elements[0].elements) {
-            if (property.name === 'w:t') {
-                return property.elements[0].text;
-            }
+    construtCiceroMarkNodeJSON(nodeInformation) {
+        let obj = {};
```

Comment:
I can infer from the method name that you are constructing a CiceroMark node. You should rename this variable accordingly.

Reply:
Look at this comment too.

Reply:
@algomaster99 
I updated it to `constructCiceroMarkNode`. Is this also not appropriate?

Reply:
I have posted this comment under `let obj = {};`.